guby_backend/views.py

```diff
@@ -138,23 +138,27 @@ def event_index(request, meetup_id):
 @has_ownership
 def event_add(request, meetup_id):
     form = EventForm(request.POST or None)
-
     if form.is_valid():
-        model = form.save(commit=False)
-        meetup = Meetup.objects.get(id=meetup_id)
-        model.meetup = meetup
-        model.status = Event.STATUS_CREATED
-        #TODO handle errors
-        model.save()
-        return redirect(f'/app/meetups/{meetup_id}/events/')
-
+        try:
+            meetup = Meetup.objects.get(id=meetup_id)
+            model = form.save(commit=False)
+            channel_name = f'{str(model.name)}_{str(random.randint(0, 10000))}'
```

Comment:
@nischalhp  @shrayasr  - Would making the channel name `<meetup.name>_<event.name>`look better?

Reply:
Yeah definitely +1

Reply:
This was exactly what I tried, somehow the rocketchat channel was not getting created, because Meetup name can have spaces in them or other weird characters, hence it will become a problem to create the channel.

Reply:
Ok then let us ship this for now. When our slug comes in (which should anyway be cleaning all this), we can revisit this :+1: